src/app/shared/menu/menu-item/link-menu-item.component.html

```diff
@@ -1,8 +1,6 @@
-
-<a class="nav-item nav-link"
+<a class="ds-menu-item" role="menuitem"
    [ngClass]="{ 'disabled': !hasLink || item.disabled }"
    [attr.aria-disabled]="!hasLink || item.disabled"
-   [title]="item.text | translate"
```

Comment:
@davide-negretti : I think this is why the e2e tests are failing in this PR.  If you look at the error in GitHub CI it says this:
> AssertionError: Timed out retrying after 4000ms: Expected to find element: `ds-navbar ds-link-menu-item a[title="Statistics"]`, but never found it.

These links no longer have a "title" text to match against.  So, the e2e tests for statistics need to be updated.  For example here: https://github.com/DSpace/dspace-angular/blob/main/cypress/e2e/collection-statistics.cy.ts#L9

I suspect the reason these e2e tests are not successful for you is because they REQUIRE the exact data from our Demo Entities Data set: https://github.com/DSpace-Labs/AIP-Files/releases/tag/demo-entities-data  If you have different data, or an expected Item/Collection/Community UUID is missing, then the e2e tests will fail.  While the demo site at https://demo.dspace.org does start with this data set, it's constantly being changed by people using the site...and those changes can result in e2e test failures.

Reply:
@tdonohue thank you, I had to remove the title to fix an accessibility issues (the title is the same as the text and it is considered redundant) but I was focusing on the class selectors. I will fix the query accordingly